protocol/0056-REWA-rewards_overview.md

```diff
@@ -403,6 +403,14 @@ Create 3 markets settling in USDT. Wait for a new epoch to begin, in the next ep
 
 Run for another epoch with no fee generated. Expect no transfer to be made to the reward pools of the accounts.
 
+### Locked Reward
+
+When a party earns locked rewards (in governance token), they are transferred into the party's locked rewards account and locked for the appropriate period before vesting.
+Sum of rewards in vesting account and locked account should count toward the party's staking balance and they can be [staked](0059-STKG-simple_staking_and_delegating.md).
```

Comment:
Small clarification, in cores implementation there are no separate locked rewards and vesting accounts, they are one single account.
```suggestion
When a party earns locked rewards (in governance token), they are transferred into the party's vesting account account but locked for the appropriate period before vesting.
The rewards in the vesting account (both locked and unlocked) should count toward the party's staking balance and they can be [staked](0059-STKG-simple_staking_and_delegating.md).
```